session-replacement/src/main/java/com/amadeus/session/SessionConfiguration.java

```diff
@@ -568,8 +581,27 @@ public void setLoggingMdcKey(String loggingMdcKey) {
   }
 
   /**
-   * Returns <code>true</code> if sessions are sticky. Sticky sessions should be
-   * expired on the last node that used them.
+   * Returns <code>true</code> if sessions id expected to be suffixed by !timestamp.
+   *
+   * @return <code>true</code> if session id to be suffixed by !timestamp.
+   */
+  public boolean isTimeStampSufix() {
+    return timestampSufix;
+  }
+
+  /**
+   * Sets session suffix by !timestamp. See {@link #isTimeStampSufix()}.
```

Comment:
How about "Add timestamp suffix to session id if set to <code>true</code>. See..."